src/main/java/org/spongepowered/api/world/storage/WorldProperties.java

```diff
@@ -105,6 +106,13 @@
     String getWorldName();
 
     /**
+     * Gets the directory the files for this world are located.
+     *
+     * @return The directory the files are located in
+     */
+    File getWorldDirectory();
+
+    /**
```

Comment:
The point wasn't that someone is working on this, merely that someone could.

Or couldn't now SpongeAPI assumes it has a directory.